src/router/index.js

```diff
@@ -17,6 +19,14 @@ export default new Router({
       path: '/biobank/report/:id',
       component: BiobankReportCard
     },
+    {
+      path: '/collection/report/:id',
+      component: CollectionReportCard
+    },
+    {
+      path: '/network/report/:id',
+      component: NetworkReportCard
+    },
```

Comment:
I see how you're continuing with the existing routes and that's probably wise given that the links can be shared. But I don't find the scheme very intuitive

Reply:
Do you suggest we keep it this way because of the sharing or should we change it?

> I see how you're continuing with the existing routes and that's probably wise given that the links can be shared. But I don't find the scheme very intuitive

Do you suggest we keep it this way because of the sharing or should we change it?

Reply:
could put a redirect on the old route and start with new names, not sure. what do you think?